src/jitlayers.cpp

```diff
@@ -539,6 +574,31 @@ CompilerResultT JuliaOJIT::CompilerT::operator()(Module &M)
                                  "The module's content was printed above. Please file a bug report");
     }
 
+    uint64_t end_time = 0;
+    if (dump_llvm_opt_stream != NULL) {
+        end_time = jl_hrtime();
+        jl_printf(dump_llvm_opt_stream, "  time_ns: %" PRIu64 "\n", end_time - start_time);
+
```

Comment:
```suggestion
        jl_printf(dump_llvm_opt_stream, "  optlevel: %d\n", optlevel);

```

Reply:
Oooh, love it. Thanks, great suggestion! :)